Mesh_3/test/Mesh_3/test_max_edge_distance.cpp

```diff
@@ -0,0 +1,279 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+#include <CGAL/Mesh_triangulation_3.h>
+#include <CGAL/Mesh_complex_3_in_triangulation_3.h>
+#include <CGAL/Mesh_criteria_3.h>
+#include <CGAL/Surface_mesh.h>
+
+#include <CGAL/Polyhedral_mesh_domain_with_features_3.h>
+#include <CGAL/Labeled_mesh_domain_3.h>
+#include <CGAL/Mesh_3/Detect_features_in_image.h>
+#include <CGAL/make_mesh_3.h>
+
+#include <CGAL/Sizing_field_with_aabb_tree.h>
+
+#include "test_meshing_utilities.h"
+
+#define CGAL_MESH_3_VERBOSE true
```

Comment:
does it make sense to have that extra log in the testsuite?

Reply:
no

Reply:
> does it make sense to have that extra log in the testsuite?

That is only for one test. Is the log that long? @sloriot, do you remember in which internal release that PR was tested? If you could give directly the link to the (long) log, that would be nice.

Reply:
I think the `\r` that display the evolving number of vertices, size of priority queue, etc, are not properly taken into account in the dump, so there is one new line for each new vertex for example

Reply:
yeah it seems it is only ~10 lines

Reply:
If you have a look at [this test page](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-6.0-Ic-245/Mesh_3_Examples/TestReport_Christo_MSVC-2022-Community-Release.gz), the output of `mesh_3D_weighted_image_with_detection_of_features` is far more than 10 lines

Reply:
> I think the `\r` that is used to display the evolving number of vertices, size of priority queue, etc, are not properly taken into account in the dump, so there is one new line for each new vertex for example

> [!TIP]
> Actually, `\r` is just a single character like any other. Whee wrote in a file (like in the logs of the testsuite), all `\r` are added to the file, and displayed later but whatever software displaying the file (like the web browser, in our case). When wrote in a terminal console, the terminal usually interprets it and move its internal writing cursor to the beginning of the line (in an overwrite mode).

Reply:
I mean `test_max_edge_distance.cpp` output is only 10 lines

Reply:
oh ok!